pkg/api/v1/assignments_delete.go

```diff
@@ -30,6 +31,25 @@ func (r *Router) assignmentsDelete(c echo.Context) error {
 			return v1InternalServerErrorResponse(c, err)
 		}
 
+		feMods := models.FrontendWhere.FrontendID.EQ(assignments[0].FrontendID)
+
+		feModel, err := models.Frontends(feMods).One(ctx, r.db)
+		if err != nil {
+			r.logger.Errorw("error fetching frontend", "error", err)
+			return v1InternalServerErrorResponse(c, err)
+		}
+
+		msg, err := pubsub.NewAssignmentMessage(someTestJWTURN, "urn:infratographer:infratographer.com:tenant:"+assignments[0].TenantID, pubsub.NewAssignmentURN(assignments[0].AssignmentID), "urn:infratographer:infratographer.com:load-balancer:"+feModel.LoadBalancerID)
```

Comment:
this is very long 😆

Reply:
There's room for improvement, for sure. I think a builder pattern maybe more towards what will make this easier in the long run.